htdocs/themes/xbootstrap/modules/lexikon/lx_bookmark.tpl

```diff
@@ -1,46 +1,46 @@
 <!-- Bookmarks -->
```

Comment:
Shouldn't `$encoded_title` default to an empty string instead of a boolean value for all the `href` links?